File: Nearly Lucky Number/Nearly_Lucky_Number.cpp
#include <iostream>
#define ll long long
using namespace std;

bool Nearly_Lucky_Number()
{
    ll number;
    cin>>number;
    int count_luck_numbers = 0;
    while(number > 0)
    { int digit = number % 10;
      if(digit == 4 || digit == 7)   //4 and 7 are lucky numbers
      count_luck_numbers++;
      number = number / 10;    
    }
    if(count_luck_numbers == 4 || count_luck_numbers == 7)
    return true;
    
    return false;

}
int main() {
	int remained_test_cases = 1 ;
// 	cin>>remained_test_cases;
	while(remained_test_cases > 0)
	{   if (Nearly_Lucky_Number())
	    cout<<"YES";
	    else
	    cout<<"NO";
	    remained_test_cases--;
	}
	return 0;
}

File: Nearly Lucky Number/Nearly_Lucky_Number.py
def Nearly_Lucky_Number():
    number = int(input())
    count_lucky_number = 0
    
    while number > 0:
        digit = number % 10
        if digit == 4 or digit == 7:    #lucky numbers are 4 and 7
            count_lucky_number += 1
        number = number // 10 
    if count_lucky_number == 4 or count_lucky_number == 7:
        return True
    return False    
            
    
    
remained_test_cases = 1 
# remained_test_cases = int(input())
while remained_test_cases > 0:
    if (Nearly_Lucky_Number()):
        print("YES")
    else:
        print("NO")
    remained_test_cases -= 1 